utils/jwtToken.js

```diff
@@ -0,0 +1,29 @@
+// Create token
+
+const sendToken = (user, statusCode, res) => {
+    const accesstoken = user.getJWTToken();
+
+    res.status(statusCode)
+    .cookie("jwt", accesstoken, {secure: true, httpOnly: true})
+    .json({
+        success: true,
+        user,
+    });
+}
+
+//Remove token
+
+const removeToken = (res) => {
+    res.status(200)
+    .cookie("jwt", null, {
+        expiresIn: new Date(Date.now())
+    }, 
+    {secure: true, httpOnly: true})
+    .json({
+        success: true,
+        message: "Logged Out"
+    });
+}
```

Comment:
Deleting the cookie should be enough. refer to this link for better understanding, https://stackoverflow.com/questions/37959945/how-to-destroy-jwt-tokens-on-logout